<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageSalariesDialog</class>
 <widget class="QDialog" name="ManageSalariesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Pay Schemes</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>621</width>
     <height>521</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Echelons</string>
    </attribute>
    <widget class="QTableView" name="tableEchelons">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>521</width>
       <height>291</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
    </widget>
    <widget class="QPushButton" name="pushNewEchel">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>350</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>New</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Base Scheme</string>
    </attribute>
    <widget class="QTableView" name="tableBase">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>521</width>
       <height>291</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBaseEchelon">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>330</y>
       <width>311</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineBaseWages">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>370</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>370</y>
       <width>61</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Wages</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>325</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Echelon</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushAddBaseScheme">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>370</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Schule</string>
    </attribute>
    <widget class="QComboBox" name="comboSchuleEchel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>330</y>
       <width>311</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>410</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Wages</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableSchule">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>521</width>
       <height>291</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineSchuleWages">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>410</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>340</y>
       <width>61</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Echelon</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboSchule">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>370</y>
       <width>311</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>374</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Courses</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushAddSchuleScheme">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>410</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Uni</string>
    </attribute>
    <widget class="QComboBox" name="comboUniEchel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>330</y>
       <width>311</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboUniCourses">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>370</y>
       <width>311</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>380</y>
       <width>61</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Courses</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineUniWages">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>410</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>340</y>
       <width>61</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Echelon</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableUni">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>521</width>
       <height>291</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>410</y>
       <width>61</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Wages</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushAddUniScheme">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>410</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Languages</string>
    </attribute>
    <widget class="QComboBox" name="comboSpracheEchel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>330</y>
       <width>311</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>380</y>
       <width>61</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Diploma</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboDiploma">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>370</y>
       <width>311</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>450</y>
       <width>61</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Wages</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineSprachWages">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>450</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>340</y>
       <width>61</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Echelon</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>521</width>
       <height>291</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushAddSpracheScheme">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>450</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>410</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Schedule</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboSchedule">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>410</y>
       <width>311</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
